### PR TITLE
Add tunables to autotune configs

### PR DESCRIPTION
**Pull request by kusumachalasani**
Signed-off-by: kusumachalasani <kchalasa@redhat.com>

### CODE REVIEW

File: manifests/autotune-configs/hotspot-micrometer-config.yaml [hunk 1]
```diff
@@ -39,7 +39,8 @@ tunables:
   - response_time
   - throughput
 
-- name: gc
+# Enable zGC if Java >=17
+- name: GCPolicy
```
Comment (dinogun) [inline manifests/autotune-configs/hotspot-micrometer-config.yaml:43]: This is currently used by the code to set the GC string and needs to be "gc"

File: manifests/autotune-configs/hotspot-micrometer-config.yaml [hunk 2]
```diff
@@ -53,24 +54,304 @@ tunables:
...
+  - response_time
+  - throughput
+  - resource_usage
+
+- name: InlineSmallCode
+  description: 'Only inline already compiled methods if their code size is less than this. Default is 1000'
+  value_type: integer
+  upper_bound: '5000'
+  lower_bound: '500'
+  step: 1
```
Comment (dinogun) [inline manifests/autotune-configs/hotspot-micrometer-config.yaml:118]: Does a higher step value make sense?

File: manifests/autotune-configs/hotspot-micrometer-config.yaml [hunk 3]
```diff
@@ -53,24 +54,304 @@ tunables:
...
+  lower_bound: '1'
+  step: 0.1
+  slo_class:
+  - response_time
+  - throughput
+  - resource_usage
+
+# ConcGCThreads dependent on ParallelGCThreads. JVM exits if ParallelGCThreads < ConcGCThreads
+- name: ConcGCThreads
+  description: 'Number of threads concurrent gc will use.'
```
Comment (dinogun) [inline manifests/autotune-configs/hotspot-micrometer-config.yaml:103]: What would be the default?

File: manifests/autotune-configs/hotspot-micrometer-config.yaml [hunk 4]
```diff
@@ -53,24 +54,304 @@ tunables:
...
+  value_type: integer
+  upper_bound: '8'
+  lower_bound: '0'
```
[the remark this answers is not in the document]
Comment (kusumachalasani) [reply manifests/autotune-configs/hotspot-micrometer-config.yaml:106]: ConcGCThreads defult value is 0. So, used the lower_bound as same.

File: manifests/autotune-configs/hotspot-micrometer-config.yaml (hunk 3 above)
Comment (kusumachalasani) [reply manifests/autotune-configs/hotspot-micrometer-config.yaml:103]: default is 0.

File: manifests/autotune-configs/hotspot-micrometer-config.yaml (hunk 2 above)
Comment (kusumachalasani) [reply manifests/autotune-configs/hotspot-micrometer-config.yaml]: updated.

File: manifests/autotune-configs/hotspot-micrometer-config.yaml (hunk 1 above)
Comment (kusumachalasani) [reply manifests/autotune-configs/hotspot-micrometer-config.yaml]: updated.

**Pull request merged**